tests/integration/data/bundle.yaml.j2

```diff
@@ -0,0 +1,55 @@
+bundle: kubernetes
+name: kubeflow-pipelines
+applications:
+  argo-controller:         { charm: ch:argo-controller, channel: latest/edge, scale: 1 }
+  metacontroller-operator: { charm: ch:metacontroller-operator, channel: latest/edge, scale: 1, trust: true }
+  minio:                   { charm: ch:minio, channel: latest/edge,       scale: 1 }
+  kfp-db:                  { charm: charmed-osm-mariadb-k8s, channel: latest/stable, scale: 1, options: { database: mlpipeline } }
```

Comment:
Can we get this bundle as close as possible to what's used right now in the Kubeflow bundle for KFP? Talking about MariaDB here.

Ideally I'd say we should just reference this bundle from the Kubeflow bundle

Reply:
Agreed, I can update it to replace mariadb with mysql-k8s.

> Ideally I'd say we should just reference this bundle from the Kubeflow bundle

That will slightly change the placement of this bundle file, but we can definitely do that. Do you mind if I add that as a secondary task after we merge this PR?

Reply:
Sure!

Reply:
I have opened an [issue](https://github.com/canonical/kfp-operators/issues/233) for discussing this. Thanks for the suggestion @kimwnasptd